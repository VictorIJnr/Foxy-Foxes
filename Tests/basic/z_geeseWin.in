6
3
6
4
3
5
3
4
3
2
3
3
3
4
3
2
4
2
4
3
3
2
4
2
3
1
3
2
4
2
3
1
4
3
3
3
3
1
4
2
0
3
0
4
4
2
3
1
5
2
4
2
