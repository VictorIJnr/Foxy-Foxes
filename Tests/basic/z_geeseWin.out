     o o o    
     o o o    
 o o o o o o o
 o o . . . o o
 . . . . . . .
     . * .    
     . . .    
Geese play. Enter move:
     o o o    
     o o o    
 o o o o o o o
 o o . . . o .
 . . . . . . o
     . * .    
     . . .    
Fox plays. Enter move:
     o o o    
     o o o    
 o o o o o o o
 o o . . . o .
 . . . * . . o
     . . .    
     . . .    
Geese play. Enter move:
     o o o    
     o o o    
 o o o . o o o
 o o . o . o .
 . . . * . . o
     . . .    
     . . .    
Fox plays. Enter move:
     o o o    
     o o o    
 o o o * o o o
 o o . . . o .
 . . . . . . o
     . . .    
     . . .    
Geese play. Enter move:
     o o o    
     o o o    
 o o o * . o o
 o o . . o o .
 . . . . . . o
     . . .    
     . . .    
Fox plays. Enter move:
     o o o    
     o o o    
 o o o . * o o
 o o . . o o .
 . . . . . . o
     . . .    
     . . .    
Geese play. Enter move:
     o o o    
     o . o    
 o o o o * o o
 o o . . o o .
 . . . . . . o
     . . .    
     . . .    
Fox plays. Enter move:
     o o o    
     o * o    
 o o o o . o o
 o o . . o o .
 . . . . . . o
     . . .    
     . . .    
Geese play. Enter move:
     o o o    
     o * o    
 o o o o . o o
 o o . o . o .
 . . . . . . o
     . . .    
     . . .    
Fox plays. Enter move:
     o o o    
     o . o    
 o o o o * o o
 o o . o . o .
 . . . . . . o
     . . .    
     . . .    
Geese play. Enter move:
     o o o    
     o . o    
 o o o o * o o
 . o . o . o .
 o . . . . . o
     . . .    
     . . .    
Fox plays. Enter move:
     o o o    
     o * o    
 o o o o . o o
 . o . o . o .
 o . . . . . o
     . . .    
     . . .    
Geese play. Enter move:
     o o o    
     o * o    
 o o o o o . o
 . o . o . o .
 o . . . . . o
     . . .    
     . . .    
Geese win!
